libs/site-base/src/containers/AddAccount/Storage/GDrive/Base.tsx

```
@@ -24,8 +25,12 @@ export const GDriveBase = ({busy, text, onAuth}: Props) => {
     setupCallback((token) => {
       const r = onAuth(token);
       if (r) {
-        r.then(() => setIsUploading(false))
-         .catch(() => setIsUploading(false))
+        r.then(() => log.info("Completed account backup to google drive"))
+         .catch(e => {
+           log.error({exception: e.message}, `Exception GDrive::onAuth: {exception} - ${e.stack}`);
+           alert("An error occured during upload.  We recommend backing up this account locally and contacting support@thecoin.io")
+         })
+         .finally(() => setIsUploading(false))
```

Eating errors is bad...